spinalcordtoolbox/utils/validation.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+# Validation related utilites
+
+
+def check_dimensions_match(input_metric, im_vertebral_labeling):
```

Comment:
Could you also add a test for this function? Say, inside a new `unit_testing/test_validation.py` module? It would be nice to verify that:

* If called with two images that have different dimensions, an error is thrown
* If called with two images that have the same dimensions, no error is thrown

For test images, you can use data from the `sct_testing_data/`. (But, I also think this is a little unclear, so I've opened #3136 so our team can discuss.)

Reply:
That's a good point, also the test data is unclear to me. After the discussion in #3136 , I will add a new unit test.

Reply:
Hi @yanchunCoBIUS. I have written some recommendations in the [Test Data](https://github.com/neuropoly/spinalcordtoolbox/wiki/Programming%3A-Test-data) wiki page. 

Could you take a look and tell me if they're clear enough? If anything is confusing or unclear, feel free to ask, and I will be happy to update things to make more sense. :slightly_smiling_face: